src/dagmc/dagmcmetadata.cpp

```diff
@@ -339,9 +341,7 @@ void dagmcMetaData::parse_tally_volume_data() {
 }
 
 std::string dagmcMetaData::to_lower(std::string input) {
-  for (int i = 0; i < input.size(); i++) {
-    input[i] = std::tolower(input[i]);
-  }
+  moab::lowercase_str(input);
```

Comment:
I don't like the use of the `moab` namespace here because one can believe this comes from MOAB where it is actually a dagmc method.

Reply:
- [ ] I don't disagree, but this happens a lot in DAGMC. In fact, the entire `DagMC` class is in the `moab` namespace. I was mostly trying to follow suit here. As an incremental step, I'll update this to a `dagmc_util` namespace. There might be an even better approach here though. I'll think on it a bit.